packages/blade/src/components/Checkbox/CheckboxGroup.stories.tsx

```diff
@@ -26,13 +34,32 @@ const Page = (): React.ReactElement => {
         CheckboxGroup can be used to group together multiple checkboxes in a forms which provides
         out of the box state management for the multi-checkboxes and other features.
       </Subtitle>
-      <Link withArrow={true} href={figmaURL} target="_blank" rel="noreferrer noopener">
-        View in Figma
-      </Link>
+      <FigmaEmbed title="Checkbox Group Figma Designs" src={figmaURL} />
       <br />
       <br />
       <Title>Usage</Title>
-      <Highlight language="tsx">{`import { CheckboxGroup } from '@razorpay/blade/components' \nimport type { CheckboxGroupProps } from '@razorpay/blade/components'`}</Highlight>
+      <Sandbox showConsole>
+        {`
+          import { CheckboxGroup, Checkbox } from '@razorpay/blade/components';
+
+          function App(): JSX.Element {
+            return (
+              <CheckboxGroup 
+                label="Which are your favorite Fruits?"
+                name="fruits" 
+                onChange={({name, values}) => console.log({name, values})}
+                helpText="Select Fruits"
+              >
+                <Checkbox value="apple">Apple</Checkbox>
```

Comment:
Since the content guidelines for Checkbox is already published, we should probably be using the prescribed examples from there instead of random fruits in our Storybook as well. @saurav12 What do you think?
@saurabhdaware [Here's the link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=15962%3A210011) to our content guidelines for Checkbox.

Reply:
Yes @nashcheez it makes sense to follow the same content on the storybook as well.

Reply:
done